samples/python/visualization_bonded.py

```diff
@@ -8,41 +8,37 @@
 import numpy
 from threading import Thread
 
-box_l = 10
-n_part = 5
+box_l = 50
+n_part = 500
 
 system = espressomd.System()
-system.time_step = 0.01
-system.skin      = 0.4
+system.time_step = 0.001
+system.cell_system.skin = 0.4
 system.thermostat.set_langevin(kT=1.0,gamma=1.0)
 
-# integration
-int_steps   = 250000
-int_n_times = 500
-
 system.box_l = [box_l,box_l,box_l]
 
 system.non_bonded_inter[0,0].lennard_jones.set_params(
     epsilon=0, sigma=1,
     cutoff=2, shift="auto")
-system.bonded_inter[0] = HarmonicBond(k=1.0,r_0=1.0)
-system.bonded_inter[1] = HarmonicBond(k=1.0,r_0=1.0)
+system.bonded_inter[0] = HarmonicBond(k=0.5,r_0=1.0)
+system.bonded_inter[1] = HarmonicBond(k=0.5,r_0=1.0)
 
 for i in range(n_part):
   system.part.add(id=i, pos=numpy.random.random(3) * system.box_l)
 
-for i in range(n_part/2):
+for i in range(n_part-1):
   system.part[i].add_bond((system.bonded_inter[0], system.part[i+1].id))
-for i in range(n_part/2,n_part-1):
-  system.part[i].add_bond((system.bonded_inter[1], system.part[i+1].id))
 
 #visualizer = visualization.mayaviLive(system)
-visualizer = visualization.openGLLive(system, {'bond_type_radius': [0.1]})
+visualizer = visualization.openGLLive(system, bond_type_radius= [0.3])
+
+system.minimize_energy.init(f_max = 10, gamma = 50.0, max_steps = 1000, max_displacement= 0.2)
+system.minimize_energy.minimize()
 
 def main():
-    for i in range(0,int_n_times):
-        print(i)
-        integrate.integrate(int_steps)
+    while True:
```

Comment:
You have removed the use of `int_n_times` and `int_steps`, so you should also remove their definitions.